x-pack/plugins/observability_solution/observability_shared/common/locators/infra/asset_details_locator.ts

```diff
@@ -13,7 +13,7 @@ export type AssetDetailsLocator = LocatorPublic<AssetDetailsLocatorParams>;
 export interface AssetDetailsLocatorParams extends SerializableRecord {
   assetType: string;
   assetId: string;
-  state?: SerializableRecord;
```

Comment:
Wasn't this necessary anymore?

Reply:
`state` isn't something that goes in the URL, we shouldn't expose it when calling `getRedirectUrl`